opendata.swiss/metadata/docker-compose.yaml

```
@@ -16,14 +16,14 @@ services:
       - OTEL_SDK_DISABLED=false
       - OTEL_SERVICE_NAME=piveau-hub-repo
       - OTEL_EXPORTER_OTLP_ENDPOINT=http://collector:4317
-      - PIVEAU_FEATURE_FLAGS=${PIVEAU_FEATURE_FLAGS}
+      #- PIVEAU_FEATURE_FLAGS=${PIVEAU_HUB_AUTHORIZATION_PROCESS_DATA}
```

Could it be possible to restore the previous behavior?


```suggestion
      - PIVEAU_FEATURE_FLAGS=${PIVEAU_FEATURE_FLAGS}
```

Or is there any issue with it?